x-pack/plugins/ml/public/application/components/anomalies_table/links_menu.tsx

```diff
@@ -298,15 +359,75 @@ export const LinksMenuUI = (props: LinksMenuProps) => {
           dataViewId === null
             ? []
             : getFiltersForDSLQuery(job.datafeed_config.query, dataViewId, job.job_id),
-      });
+        ...(withWindowParameters
+          ? {
+              wp: { bMin, bMax, dMin, dMax },
+            }
+          : {}),
+      };
+    };
+
+    const generateDiscoverUrl = async () => {
+      const pageState = await generateRedirectUrlPageState();
+      const url = await discoverLocator.getRedirectUrl(pageState);
 
       if (!unmounted) {
         setOpenInDiscoverUrl(url);
       }
     };
 
+    const generateLogRateAnalysisUrl = async () => {
+      if (
+        !LOG_RATE_ANALYSIS_ML_FUNCTIONS.includes(
+          props.anomaly.source.function as LogRateAnalysisMlFunctions
```

Comment:
Can you just check `function_description === 'count'` here rather than creating `LOG_RATE_ANALYSIS_ML_FUNCTIONS` to check against `function`?

Reply:
This check makes sure we support the different variants of `count` like `high_count`, `low_count` etc.? Are you suggesting we only should support `count`? Is `function_description` not a freeform field a user could change or am I misinterpreting the field?

Reply:
No, `function_description` should return `count` for all the variants of the count detector. This field is returned by the backend and isn't editable as far as I know. We already make use of this for example in deciding whether to display [actual and typical](https://github.com/elastic/kibana/blob/main/x-pack/packages/ml/anomaly_utils/anomaly_utils.ts#L260) values for an anomaly.

Reply:
That's great to know! I changed the code to check for `function_description` in 
6c5cd53.